app.rb

```diff
@@ -0,0 +1,143 @@
+require 'pry'
+
+class Ocean
+    attr_accessor :board, :first_ship_row, :row
+
+    def initialize()
+        @@board = [
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false],
+            [false, false, false, false, false, false, false, false, false, false]
+         ]
```

Comment:
Trying to think of the laziest/clearest way to write this...

```rb
Array.new 10, false # generates an array of 10 `false`s
```
So therefore...
```rb
Array.new 10, (Array.new 10, false)
```
would give you your grid, but that's also maybe not the most readable...